OpenRA.Mods.Common/Projectiles/LaserZap.cs

```diff
@@ -128,7 +131,7 @@ public void Tick(World world)
 		{
 			// Beam tracks target
 			if (info.TrackTarget && args.GuidedTarget.IsValidFor(args.SourceActor))
-				target = args.GuidedTarget.Positions.PositionClosestTo(source);
+				target = info.TargetCenterPosition ? args.GuidedTarget.CenterPosition : args.GuidedTarget.Positions.PositionClosestTo(args.Source);
```

Comment:
'source' was changed to 'args.Source' here. The remaining 1 place where source is also used (except in ctor after assignment to source) is in following line 138. It would be better to use in both cases the same field inside one function.